resqpy/grid/grid.py

```diff
@@ -505,7 +505,7 @@ def off_handed(self):
         """Returns False if IJK and xyz have same handedness, True if they differ."""
 
         ijk_right_handed = self.extract_grid_is_right_handed()
-        assert rqet.find_tag_text(self.crs_root, 'ProjectedAxisOrder').lower() == 'easting northing'
+        assert self.crs.axis_order.lower() == 'easting northing'
```

Comment:
I'd like to see a defensive call to the self.set_crs() method in each function that uses self.crs.  This is to cater for ad hoc construction of a new Grid object, when the crs is not necessarily established by the init method.